packages/scene-composer/src/store/slices/SceneDocumentSlice.ts

```diff
@@ -384,30 +383,26 @@ export const createSceneDocumentSlice = (set: SetState<RootState>, get: GetState
           draft.document.properties = {};
         }
         draft.document.properties[property] = value;
-        draft.lastOperation = 'setSceneProperty';
-      });
-    },
 
-    clearTemplatizedDataBindings: () => {
-      set((draft) => {
-        Object.values(draft.document.nodeMap)
-          .flatMap((node) => node.components)
-          .filter(
-            (component) =>
-              component.type === KnownComponentType.Tag || component.type === KnownComponentType.ModelShader,
-          )
-          .forEach((component) => {
-            const dataBindingContext = (component as IDataBoundSceneComponentInternal)?.valueDataBinding;
-            const hasDataBindingTemplate =
-              Object.values((dataBindingContext?.dataBindingContext as any) ?? {}).filter((value) =>
-                isDataBindingTemplate(value as string),
-              ).length > 0;
-            if (hasDataBindingTemplate) {
-              dataBindingContext!.dataBindingContext = {};
-            }
-          });
-
-        draft.lastOperation = 'clearTemplatizedDataBindings';
+        if (property === KnownSceneProperty.DataBindingConfig) {
+          Object.values(draft.document.nodeMap)
+            .flatMap((node) => node.components)
```

Comment:
we actually shouldn't filter by component type but just check if the component has data binding.. this is not scalable since we will keep adding new component types with data binding, and it's hard to know we need to add the type here (which is the situation we are currently at...)